goth/runner/web_server.py

```diff
@@ -53,7 +53,7 @@ async def start(self, server_address: str) -> None:
         self._server_task = asyncio.create_task(site.start())
         logger.info(
             "Web server listening on %s:%s, root dir is %s",
-            server_address,
+            server_address or "*",
```

Comment:
Nitpicking again, but isn't this `0.0.0.0`? :thinking:

Reply:
@zakaprov given that `0.0.0.0` is not a real address, I guess it's a matter of convention ... afaik, e.g. netstat shows an asterisk but other tools may show the zeroes ... I would leave that as an asterisk but am not really polarized here...

Reply:
Alright, I guess I'm more used to seeing `0.0.0.0` myself, let's leave it as is.